src/modules/app/containers/I18nRedirect.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { replace } from 'redux-little-router'
+import normalizeUrl from 'normalize-url'
+
+import { connect } from 'react-redux'
+
+export class I18nRedirect extends React.Component {
+  static propTypes = {
+    redirect: PropTypes.func.isRequired,
+    currentPath: PropTypes.string.isRequired
+  }
+
+  static contextTypes = {
+    i18n: PropTypes.object.isRequired
```

Comment:
We shouldn't access the context directly. Is there any other method to do that?

Reply:
Maybe we could add the i18nLanguage into the store in our i18nextProvider?

Reply:
change PropTypes.object to PropTypes.instanceOf(i18n)

Reply:
Type of i18n is actually object. instanceOf does not work.

@Schedulaar why shouldn't we access the context? This is how the translate hoc works.